server/api.go

```diff
@@ -503,7 +503,7 @@ func extractClientAddressFromContext(logger *zap.Logger, ctx context.Context) (s
 
 func extractClientAddressFromRequest(logger *zap.Logger, r *http.Request) (string, string) {
 	var clientAddr string
-	if ips := r.Header.Get("x-forwarded-for"); len(ips) > 0 {
+	if ips := r.Header.Get("X-Real-Ip"); len(ips) > 0 {
```

Comment:
I'm hesitant of changing the complete header for this detection, the `x-forwarded-for` header should work, but I think that there's a bug, we are returning the first ip and we should always return the last one for that.
If we still want this, we should remember down the road that we did this change.